### PR TITLE
Mnj 18 complementary feeding

### PR DESCRIPTION
**Pull request by issyzac**

### CODE REVIEW

**Comment by devcolonel** [comment]
@issyzac after testing I found that we are supposed to add/register the recurring service on **nacp flavour** 
servive.json
recurring_service_types.json

File: opensrp-chw/src/main/assets/services.json
Comment (devcolonel) [inline opensrp-chw/src/main/assets/services.json:1]: Complimentary feeding service needs to be registered on the nacp flavour

File: opensrp-chw/src/main/assets/recurring_service_types.json
Comment (devcolonel) [inline opensrp-chw/src/main/assets/recurring_service_types.json:1]: Complimentary feeding service needs to be added on the nacp flavour

Comment (issyzac) [reply opensrp-chw/src/main/assets/recurring_service_types.json:1]: Technically it should work fine whether under the main package or under a specific flavor so on testing its not expected to have outcomes that are different but for standardization sake I agree its a good practise to keep it under a specific flavor

**Review by gosso22 — APPROVED** [review]
LGTM!

**Pull request merged**